shell/platform/fuchsia/flutter/gfx_external_view_embedder.cc

```diff
@@ -328,15 +328,13 @@ void GfxExternalViewEmbedder::SubmitFrame(
             kScenicZElevationBetweenLayers * scenic_layer_index +
             embedded_views_height;
 
-        if (view_mutators.total_transform != view_params.transformMatrix()) {
-          FML_LOG(ERROR) << "Failed assertion: view_mutators.total_transform "
-                            "!= view_params.transformMatrix()";
-          FML_LOG(ERROR) << "view_mutators.total_transform:";
-          view_mutators.total_transform.dump();
-          FML_LOG(ERROR) << "view_params.transformMatrix():";
-          view_params.transformMatrix().dump();
-          FML_LOG(FATAL) << "view_mutators.total_transform "
-                            "!= view_params.transformMatrix()";
+        // Use built-in get method for SkMatrix to get values
+        // See:
+        // https://source.corp.google.com/piper///depot/google3/third_party/skia/HEAD/include/core/SkMatrix.h;l=391
+        for (int index = 0; index < 9; index++) {
```

Comment:
Let's surround this in `#ifdef NDEBUG` to make it a debug only assertion. I don't see any reason to do this on every frame in a release build.